pom.xml

```diff
@@ -25,7 +25,9 @@
         </developer>
     </developers>
     <properties>
-        <workflow.version>1.3</workflow.version>
+        <jenkins.version>1.580.1</jenkins.version>
+        <java.level>6</java.level>
+        <workflow.version>1.4.3-beta-1</workflow.version>
```

Comment:
Well the wiki page for each 2.x plugin includes a changelog. For 1.x maintenance releases, the existing `CHANGELOG.md` may be used. I did not bother mentioning `1.4.3-beta-1` since it is intended for test dependencies, not users.